Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
         patch pkg-config swig unzip wget xz-utils \
         autoconf autotools-dev automake texinfo dejagnu \
         build-essential libtool libltdl-dev \
-        gnupg2 libdbus-glib-1-2 sudo sqlite3 \
+        gnupg2 libdbus-glib-1-2 sudo sqlite3 cmake \
```

Comment:
pyodide-build depends on `cmake` PyPI package, so one should be available to use cmake after installing pyodide-build. I wonder why cmake is necessary here.

Reply:
I think I added it because of the error we were getting in https://github.com/pyodide/pyodide/pull/5510.